LibraBFT/Impl/Consensus/BlockStorage/BlockTree.agda

```diff
@@ -89,11 +89,11 @@ insertQuorumCertE qc bt0 = do
 
   continue1 : BlockTree → HashValue → ExecutedBlock → List InfoLog → (BlockTree × List InfoLog)
   continue1 bt blockId block info =
-    continue2 ( record bt { _btIdToQuorumCert = lookupOrInsert blockId qc (bt ^∙ btIdToQuorumCert) } )
+    continue2 ( bt & btIdToQuorumCert ∙~ lookupOrInsert blockId qc (bt ^∙ btIdToQuorumCert))
               ( (fakeInfo ∷ info) ++ (if ExecutedBlock.isNilBlock block then fakeInfo ∷ [] else [] ))
   continue2 bt info =
     if-dec (bt ^∙ btHighestCommitCert ∙ qcCommitInfo ∙ biRound) <? (qc ^∙ qcCommitInfo ∙ biRound)
-    then (record bt { _btHighestCommitCert = qc } , info)
+    then ((bt & btHighestCommitCert ∙~ qc) , info)
```

Comment:
There are more parens than necessary (and inconsistent with the Haskell code) but otherwise looks good.

Reply:
Those parens are necessary to compile.